lib/gobl/num/amount.rb

```diff
@@ -209,7 +207,7 @@ def parse(str)
           v2 = x[1].to_i
           e = x[1].length
           v *= 10**e
-          v += v.positive? ? v2 : -v2
+          v += x[0].match?(/^-/) ? -v2 : v2
```

Comment:
Perhaps String#start_with? (https://apidock.com/ruby/String/start_with%3F) would be more efficient?

Reply:
Oh, sure! I don't know how I didn't think of it. I'll change it.